src/lib.rs

```diff
@@ -90,9 +45,9 @@ impl<T: Copy + PartialOrd + Debug> CKMS<T> {
     /// ```
     /// use quantiles::CKMS;
     ///
-    /// let mut ckms = CKMS::<u16>::new(0.001);
+    /// let mut ckms = CKMS::<u64>::new(0.001);
```

Comment:
`Iterator::fold` ought to allow us to sum over floats, but the docs aren't very clear wrt overflow behavior. thoughts?